.github/workflows/validate.yml

```diff
@@ -5,11 +5,23 @@ on:
       - main
 jobs:
   build:
-    name: Build
+    name: Build and test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '3.1.x'
-      - run: dotnet build
+      - name: Build
+        run: dotnet build
+      - name: Test
+        env:
+          BW_ACCOUNT_ID: ${{ secrets.BW_ACCOUNT_ID }}
+          BW_USERNAME: ${{ secrets.BW_USERNAME }}
+          BW_PASSWORD: ${{ secrets.BW_PASSWORD }}
+          BW_VOICE_APPLICATION_ID: ${{ secrets.BW_VOICE_APPLICATION_ID }}
+          BW_MESSAGING_APPLICATION_ID: ${{ secrets.BW_MESSAGING_APPLICATION_ID }}
+          BW_NUMBER: ${{ secrets.BW_NUMBER }}
+          USER_NUMBER: ${{ secrets.USER_NUMBER }}
```

Comment:
I guess `USER_NUMBER` is also a BW number? I prefer the inbound/outbound terms

Reply:
I used the same variable names to be consistent with the samples template and not require a second set of variable names for testing.

Reply:
I guess we can go with BW == outbound and USER == inbound